<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogBlogDialog</class>
 <widget class="QDialog" name="LogBlogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="gboxTop">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QCheckBox" name="ckTriggerTime">
                <property name="text">
                 <string>时间</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="labTimeStart">
                <property name="text">
                 <string>开始时间</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QDateEdit" name="dateStart">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::WheelFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::DefaultContextMenu</enum>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhPreferNumbers</set>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="accelerated">
                 <bool>false</bool>
                </property>
                <property name="keyboardTracking">
                 <bool>true</bool>
                </property>
                <property name="displayFormat">
                 <string>yyyy年M月d日</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="labTimeEnd">
                <property name="text">
                 <string>结束时间</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDateEdit" name="dateEnd">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="displayFormat">
                 <string>yyyy年M月d日</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="ckTriggerType">
                <property name="text">
                 <string>类型</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cboxTriggerType">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>顾客</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>管理员</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="ckUserName">
                <property name="text">
                 <string>操作员</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cboxUserName">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>admin</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>管理员</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QPushButton" name="btnSelect">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>查询(&amp;S)</string>
                </property>
                <property name="icon">
                 <iconset resource="image.qrc">
                  <normaloff>:/image/select.png</normaloff>:/image/select.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="btnDelete">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>删除(&amp;D)</string>
                </property>
                <property name="icon">
                 <iconset resource="image.qrc">
                  <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnPrint">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>打印(&amp;P)</string>
                </property>
                <property name="icon">
                 <iconset resource="image.qrc">
                  <normaloff>:/image/print.png</normaloff>:/image/print.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="btnExcel">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>功能开发中</string>
                </property>
                <property name="text">
                 <string>导出(&amp;E)</string>
                </property>
                <property name="icon">
                 <iconset resource="image.qrc">
                  <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>操作日志</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
